src/SixLabors.Fonts/Tables/General/VerticalMetricsTable.cs

```diff
@@ -47,7 +47,10 @@ internal short GetTopSideBearing(int glyphIndex)
             return null;
         }
 
-        return Load(binaryReader, headTable.NumberOfVMetrics, profileTable.GlyphCount);
+        using (binaryReader)
```

Comment:
Nice job finding this! Did you find it profiling or just by chance?

Reply:
I've used [Roslyn profiler](https://github.com/DotNetAnalyzers/IDisposableAnalyzers), there are some other issues, but due to used patterns in Fonts/ImageSharp I still investigating if it can be fixed (there are some non-sealed classes with non-virtual Dispose and some classes has a local variables, where class is not an owner of it). For. ex., `FontReader` class has a variable `stream`, but it has no Dispose, but some other classes implements it

Reply:
I'd be careful with that analyzer, I've seen many, many false positives. 

`FontReader` doesn't own the input stream so shouldn't be responsible for its lifecycle. It's perfectly fine for the type to use a private variable to hold the reference.

Reply:
Yes, it'll should be used carefully, because of such FPs

Reply:
I'll prepare some interesting cases and will add an appropriate discussion if needed, still need some time to research all warning...